requirements/extra-ml-requirements.txt

```diff
@@ -11,8 +11,7 @@ fastai>=2.4.1
 # Required by mlflow.spacy
 spacy>=3.3.0
 # Required by mlflow.tensorflow
-# TODO: Unpin upper bound once transformers releases 2.11 compatibility fix
-tensorflow>=2.8.0,<2.11.0
```

Comment:
No need to pin here if we disable the test case on TF 11

Reply:
The only downside to removing the pin is if a contributor or user installs a fresh environment from our requirements files and tries to use TF. Probably not going to happen though.